### ISSUE TITLE
Web3 hybrid game integration

### ISSUE THREAD

**Issue by hieudd** [Koniverse/Playnation#40]
**Work with current Polkadot meme games to integrate into Playnation**

From the game publisher’s perspective, Playnation will be the platform where Web2 games approach Web3 users and convert their games to Web3 step by step.

One of the jobs we need to do is to help Polkadot's current game projects reach more players. For example:

- [ ] Pink Drop
- [ ] DED Mine
- [ ] Flappy Wud
- [ ] etc.

## Feasibility research:
Check if in the mobile environment, it is possible to connect the wallet in the form of Wallet Connect ⇒ request to sign transactions and messages through Wallet Connect as well as check some corresponding information.

**Technical notes:**

Need to link an account with a single wallet address

- Wallet link API ⇒ needs to have rules regulating linking to prevent fraud
- Authenticate the link through signature
- Some elements such as tasks... will be checked through the linked wallet address
- Added a number of mechanisms that allow on-chain implementation directly on the telegram wallet or linked wallet

Basically, this feature is technically feasible but will require further evaluation and calculations in terms of usage to ensure a seamless experience as well as a clear estimate of the complexity of implementation.
